### DISCUSSION TITLE
DAG not shown in UI

### DISCUSSION THREAD

**Discussion by scheung38** [apache/airflow#14169] [Q&A] [answered]
Can someone point out why, DAGS as project/dags/hello_world.py does not appear in Airflow UI?

```
from datetime import datetime
from airflow import DAG
from airflow.operators.python_operator import PythonOperator
from icecream import ic

default_args = {'owner': 'airflow', 'start_date': datetime(2021, 1, 1)}

dag = DAG('hello_world', schedule_interval='daily', default_args=default_args, catchup=False)

def hello_world_py():
    ic('Hello World')

with dag:
    t1 = PythonOperator(
        task_id='hello_world',
        python_callable=hello_world_py)
    )
```

i.e UI with empty tasks 

thanks

**Comment by kaxil** [comment, marked as answer]
The `schedule_interval` should be `@daily`.

Other than that everything else looks good. What do you see in the UI?

**Reply by scheung38** [reply]
Thanks @kaxil, trying to decide whether to use apache-airflow or from astronomer-airflow for on-prem, but just like to know what astronomer-airflow provides out of the box (without the Enterprise plan) that apache is lacking? Any ideas?

**Reply by banderson131** [reply]
Hi @scheung38 - I can share more about that with you and go into more detail around the local development, logging, monitoring, and operational aspects of Astronomer.
